apps/src/StudioApp.js

```diff
@@ -251,6 +251,15 @@ StudioApp.prototype.init = function(config) {
     config = {};
   }
 
+  if (config.isLegacyShare && config.controllerAction == 'show') {
```

Comment:
discussed offline. I'll leverage config.hideSource and don't need to add another config value